usaspending_api/submissions/management/commands/load_submission.py

```diff
@@ -270,30 +257,35 @@ def handle(self, *args, **options):
             location = load_data_into_model(Location(), row, field_map=place_of_performance_field_map, value_map=pop_value_map, as_dict=True)
             pop_location, created = Location.objects.get_or_create(**location)
 
-            # Create the base award, create actions, then tally the totals for the award
+            # Try and find the parent award
+            award = None
+            try:
+                award = find_or_create_summary_award(piid=row.get('piid'), fain=row.get('fain'), uri=row.get('uri'), parent_award_id=row.get('parent_award_id'))
```

Comment:
Same question as below about the use of `parent_award_id` when looking for the summary award.